nav2_simple_navigator/src/simple_navigator.cpp

```diff
@@ -36,62 +35,55 @@ SimpleNavigator::~SimpleNavigator()
 }
 
 nav2_lifecycle::CallbackReturn
-SimpleNavigator::on_configure(const rclcpp_lifecycle::State & state)
+SimpleNavigator::on_configure(const rclcpp_lifecycle::State &)
```

Comment:
I believe there's been a consistent feedback on `\*...*\` for unused parameters

Reply:
Yup. Missed a few here and there. Update.